### PR TITLE
Update abilities.rst

### PR DESCRIPTION
**Pull request by v-h-gn**
_Use the following template. Feel free to remove any unused sections._

### Elements and Abilities
* Changes `` Each of the elements and chiblocking have various abilities which can be used for combat, exploration, and resource gathering.`` to ``Elements contain abilities which the player can use to aid themself. ``
* Adds ``Below are outlines of each element and the abilities they contain.``
> * https://trello.com/c/hyhX11bk/67-grammar-fixes

### Airbending
* Change `` on movement and speed, but also have a variety of combat techniques.``  to `` on movement and speed.``
> * https://trello.com/c/hyhX11bk/67-grammar-fixes

### Earthbending
* Change ``on strengthening and defending oneself while remaining on the ground when possible`` to ``on strengthening and defending oneself.``
> * https://trello.com/c/hyhX11bk/67-abilitiesrst-grammar-fixes

### Waterbending
* Change ``able to adapt between offense, defense, and mobility.`` to ``on shifting between offense and defense.``
> * https://trello.com/c/hyhX11bk/67-abilitiesrst-grammar-fixes

### Firebending
* Changes ``onslaught of offensive attacks instead of being defensive.`` to  ``onslaught of offensive attacks.``
* Changes ``Firebending abilities focus on overwhelming combat potential and survival.`` to ``Firebending abilities focus on combat capability and destroying obstacles.``
> * https://trello.com/c/hyhX11bk/67-abilitiesrst-grammar-fixes

### Chiblocking
* Changes `` in bender's bodies, doing anything from disabling their bending to temporarily paralyzing
them.`` to ``in bender's bodies disabling their bending.``
> * https://trello.com/c/hyhX11bk/67-abilitiesrst-grammar-fixes

### CODE REVIEW

**Review by jphuse — DISMISSED** [review]
This looks good to me.

File: source/pkcore/abilities.rst [hunk 1]
```diff
@@ -1,17 +1,16 @@
 ======================
 Elements and Abilities
 ======================
-Each of the elements and chiblocking have various abilities which
-can be used for combat, exploration, and resource gathering.
+Elements contain abilities which the player can use to aid themself.
```
Comment (ChristopherWMM) [inline source/pkcore/abilities.rst]: I think keeping some of the original would be good. Potentially:

> Each element has various abilities which can be used for combat, exploration, and gathering resource.

File: source/pkcore/abilities.rst [hunk 2]
```diff
@@ -52,10 +50,9 @@ Waterbending
 ============
 Water is the element of change. Waterbenders use their plethora of
 skills to redirect and manipulate opponents attacks against them.
-Similar to Earthbenders, they employ both defensive and offensive
-strategies to accomplish this.
 
-Waterbending abilities focus on being able to adapt between offense, defense, and mobility.
+
+Waterbending abilities focus on shifting between offense and defense.
```
Comment (ChristopherWMM) [inline source/pkcore/abilities.rst]: > Waterbending abilities focus on being able to shift between offense, defense, and mobility.

File: source/pkcore/abilities.rst [hunk 3]
```diff
@@ -66,14 +63,12 @@ Waterbending abilities focus on being able to adapt between offense, defense, an
 Chiblocking
 ===========
 Chiblocking is the anti-bending fighting style. Chiblockers use critically
-placed kicks and punches to block chi paths in bender's bodies, doing
-anything from disabling their bending to temporarily paralyzing
-them.
+placed kicks and punches to block chi paths in bender's bodies disabling their bending.
 
-Chiblocking abilities are purely for combat, being able to temporarily block a bender's chi so they are unable to bend.
+Chiblocking abilities focus on countering bending abilities.
```
Comment (ChristopherWMM) [inline source/pkcore/abilities.rst]: > Chiblocking abilities focus on close quarters combat and countering bending abilities.

File: source/pkcore/abilities.rst [hunk 4]
```diff
@@ -38,9 +36,9 @@ Firebending
 ===========
 Fire is the element of power. Firebenders are very aggressive and
 powerful, choosing to overwhelm their opponents with an endless
-onslaught of offensive attacks instead of being defensive.
+onslaught of offensive attacks.
 
-Firebending abilities focus on overwhelming combat potential and survival.
+Firebending abilities focus on combat capability and destroying obstacles.
```
Comment (ChristopherWMM) [inline source/pkcore/abilities.rst]: > Firebending abilities focus on overwhelming combat potential and destroying obstacles.

File: source/pkcore/abilities.rst [hunk 5]
```diff
@@ -1,17 +1,16 @@
...
 
 Airbending
 ==========
 Air is the element of freedom. Airbenders are quick, agile, and
 passive in nature, preferring to bob and weave through flurries of
 attacks, and defending if they have no other option.
 
-Airbending abilities tend to focus on movement and speed, but also
-have a variety of combat techniques.
+Airbending abilities focus on movement and speed.
```
Comment (ChristopherWMM) [inline source/pkcore/abilities.rst]: > Airbending abilities focus on fast mobility and a variety of combat techniques.

**Review by ChristopherWMM — CHANGES_REQUESTED** [review]

File: source/pkcore/abilities.rst [hunk 6]
```diff
@@ -66,14 +63,12 @@ Waterbending abilities focus on being able to adapt between offense, defense, an
 Chiblocking
 ===========
 Chiblocking is the anti-bending fighting style. Chiblockers use critically
-placed kicks and punches to block chi paths in bender's bodies, doing
-anything from disabling their bending to temporarily paralyzing
-them.
+placed kicks and punches to block chi paths in bender's bodies disabling their bending.
 
-Chiblocking abilities are purely for combat, being able to temporarily block a bender's chi so they are unable to bend.
+Chiblocking abilities focus on close quarters combat countering bending abilities.
```
Comment (ChristopherWMM) [inline source/pkcore/abilities.rst]: > Chiblocking abilities focus on close quarters combat and countering bending abilities.

Comment (ChristopherWMM) [reply source/pkcore/abilities.rst]: Chiblockers deal a lot of damage melee regardless of whether the target is a bender or not. I think that having that separation is important.

File: source/pkcore/abilities.rst [hunk 7]
```diff
@@ -52,10 +50,9 @@ Waterbending
 ============
 Water is the element of change. Waterbenders use their plethora of
 skills to redirect and manipulate opponents attacks against them.
-Similar to Earthbenders, they employ both defensive and offensive
-strategies to accomplish this.
 
-Waterbending abilities focus on being able to adapt between offense, defense, and mobility.
+
+Waterbending abilities focus on rapidly shifting between mobility, offense and defense.
```
Comment (ChristopherWMM) [inline source/pkcore/abilities.rst]: Oxford comma please ❤️ 

> Waterbending abilities focus on rapidly shifting between mobility, offense, and defense.

File: source/pkcore/abilities.rst [hunk 8]
```diff
@@ -1,17 +1,16 @@
 ======================
 Elements and Abilities
 ======================
-Each of the elements and chiblocking have various abilities which
-can be used for combat, exploration, and resource gathering.
+Each element contains a variety of abilities that the player can use to aid their survival.
```
Comment (ChristopherWMM) [inline source/pkcore/abilities.rst]: > Each element has a variety of abilities that can be used for combat, exploration, and gathering resource.

**Review by ChristopherWMM — CHANGES_REQUESTED** [review]

File: source/pkcore/abilities.rst (hunk 6 above)
Comment (vtovmasian) [reply source/pkcore/abilities.rst]: oop typo

**Review by ChristopherWMM — APPROVED** [review]
Looks good 👍

**Review by jphuse — APPROVED** [review]

**Pull request merged**